src/pages/public_registration/PublicGeneralContent.tsx

```diff
@@ -80,7 +80,7 @@ export const PublicGeneralContent = ({ registration }: PublicRegistrationContent
         <Typography>{displayDate(entityDescription?.date)}</Typography>
 
         {journalPublicationInstance?.contentType && (
-          <Typography>{t(`resource_type.content_types.${journalPublicationInstance?.contentType}`)}</Typography>
+          <Typography>{t(`resource_type.content_types.${journalPublicationInstance?.contentType}`, '')}</Typography>
```

Comment:
mulig fallbacken kan fjernes når ny datamodell rulles ut. Dobbeltsjekk med denne registreringen som har lagret contentType på gammelt format: https://frontend.dev.nva.aws.unit.no/registration/01814df46484-3f1705ae-fe85-409a-9279-bbc1853662f6
```suggestion
          <Typography>{t(`resource_type.content_types.${journalPublicationInstance?.contentType}`)}</Typography>
```